recipes/h5pp/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+import os, re
+
+class h5ppConan(ConanFile):
+    name = "h5pp"
+    version = "1.8.2"
+    description = "A C++17 wrapper for HDF5 with focus on simplicity"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/DavidAce/h5pp"
+    topics = ("conan","h5pp","hdf5", "binary", "storage")
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    requires = "eigen/3.3.7", "spdlog/1.7.0", "hdf5/1.12.0"
+    exports_sources = ["CMakeLists.txt"]
+
+    options = {
+        'tests'     :[True,False],
+        'examples'  :[True,False],
+        'verbose'   :[True,False],
+        }
+
+    default_options = (
+        'tests=False',
+        'examples=False',
+        'verbose=False',
+    )
+
+    _cmake = None
```

Comment:
I think these options can be removed completely.
These options won't change the final package.
They will just change the build process.

```suggestion
    exports_sources = ["CMakeLists.txt"]

    _cmake = None
```